src/gui/editors/PianoRoll.cpp

```diff
@@ -4745,9 +4782,10 @@ PianoRollWindow::PianoRollWindow() :
 	eraseAction->setShortcut( Qt::SHIFT | Qt::Key_E );
 	selectAction->setShortcut( Qt::SHIFT | Qt::Key_S );
 	pitchBendAction->setShortcut( Qt::SHIFT | Qt::Key_T );
-
-	connect( editModeGroup, SIGNAL( triggered( int ) ), m_editor, SLOT( setEditMode( int ) ) );
-
+	connect( editModeGroup, SIGNAL( triggered( int ) ), m_editor, SLOT( setEditMode( int ) ) );		
```

Comment:
When you add or change code, try to use the [new code style](https://github.com/LMMS/lmms/wiki/Coding-Conventions) ... that means no spaces around ( )

Reply:
Oh.. a new style. Ok.
I will go over all functions for that too.
Does that mean that all code need to be overhauled as well? 
Where has this decision been taken. I try to keep track of things like that, because it is annoying work to make code-style-changes, but i havent seen this new conformity-style change in any posts. We (properly) should have a special page in dev-wiki for things like that, but i will fix it.

Reply:
The white space rule has existed since 2019 when the Coding-Conventions page was created, so that's probably why you never heard any discussion about changing it.

In general, only change the coding style on the lines you edit. That makes it easier to see which lines has been changed when people are reviewing your PR, and it creates less conflicts.

Reply:
> In general, only change the coding style on the lines you edit. That makes it easier to see which lines has been changed when people are reviewing your PR, and it creates less conflicts.

👍Got it! Will do.